README.yaml

```diff
@@ -156,9 +156,10 @@ examples: |-
   Here's a stub you can include into a `Makefile` to make it easier to install binary dependencies.
 
   ```
+  THIS_PATH := $(abspath $(patsubst %/,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST))))))
```

Comment:
Can you give some more context on why this is necessary? What problem did this solve for you?

Reply:
It feels like it makes an assumption about where the packages are checked out relative to `MAKEFILE_LIST`. I think for an example, it remains more clear to just say "look, suppose you have a `PACKAGES_PATH` somewhere, this is how you can invoke it" rather than say you have to check it out to `./`

Case in point, we check it out somewhere else in our `cloudposse/build-harness`

Reply:
The default example makefile for Makefile inclusion did not work for me without fully qualifying the path for INSTALL_PATH. Using the (overcomplicated) THIS_PATH local path detection may have been overkill but it got the example to work as intended. I maybe missed the point though (that has been known to happen :) )

I'm grateful for the entire suite of cloudposse's releases and am glad I'm not the only person insane enough to write 1000's of lines of Makefile code. Thanks for all of the awesome tooling and other materials.

```bash
[9:48:57] zloeber in ~/Projects/Temp ❯ make packages/install/helmfile
Installing packages master...
Cloning into 'packages'...
remote: Enumerating objects: 347, done.
remote: Counting objects: 100% (347/347), done.
remote: Compressing objects: 100% (187/187), done.
remote: Total 347 (delta 73), reused 227 (delta 48), pack-reused 0
Receiving objects: 100% (347/347), 65.70 KiB | 21.90 MiB/s, done.
Resolving deltas: 100% (73/73), done.
make[1]: Entering directory '/home/zloeber/Projects/Temp/packages/install'
make[2]: Entering directory '/home/zloeber/Projects/Temp/packages/vendor/helmfile'
curl --retry 3 --retry-delay 5 --fail -sSL -o packages//vendor/helmfile https://github.com/roboll/helmfile/releases/download/v0.90.8/helmfile_linux_amd64 && chmod +x packages//vendor/helmfile
curl: (23) Failed writing body (0 != 16360)
Makefile:10: recipe for target 'install' failed
make[2]: *** [install] Error 23
make[2]: Leaving directory '/home/zloeber/Projects/Temp/packages/vendor/helmfile'
Makefile:9: recipe for target 'helmfile' failed
make[1]: *** [helmfile] Error 2
make[1]: Leaving directory '/home/zloeber/Projects/Temp/packages/install'
Makefile:16: recipe for target 'packages/install/helmfile' failed
make: *** [packages/install/helmfile] Error 2
```

Reply:
Aha! Thanks for sharing that. I think this is the real culprit:

```
curl --retry 3 --retry-delay 5 --fail -sSL -o packages//vendor/helmfile https://github.com/roboll/helmfile/releases/download/v0.90.8/helmfile_linux_amd64 && chmod +x packages//vendor/helmfile
curl: (23) Failed writing body (0 != 16360)
```
Which is just really saying that `packages//vendor/` does not exist.

See: 
https://github.com/cloudposse/packages/blob/master/tasks/Makefile.package#L33

Note: all the other macros have a `mkdir -p`. This one does not. Let's try fixing the macro first?

Reply:
I saw that and updated to eliminate the `//` but still ran into issues. I think calling make recursively will require a full path for INSTALL_PATH regardless. I'm still groking the logic used for this project so please forgive me if I'm missing fundamental logic though.

I've personally used the following without issues:

`make packages/install/helmfile INSTALL_PATH=~/.local/bin` 

Perhaps simply adding a usage example like this would be sufficient?

Reply:
Okay, I've opened up #272 to fix the example. I think this is the minimal change.